packages/xo-web/src/xo-app/home/pool-item.js

```diff
@@ -32,7 +32,7 @@ import styles from './index.css'
     getPoolHosts,
     hosts => {
       return Promise.all(map(hosts, host => getHostMissingPatches(host))).then(
-        patches => uniq(map(flatten(patches), 'name'))
+        patches => uniq(map(flatten(compact(patches)), 'name'))
```

Comment:
Why not use `Array#filter(Boolean)` instead?

Reply:
For the sake of using native methods? Because that's pretty much what `_.compact` does, right?

Reply:
Exactly, but I think it does not have enough benefits to make it worth being used here.

Benefits of using native method:
- no external dependency
- usually better known, it's easier to see what it does, for instance, I had to check `lodash` doc here.